example/glue/configure_includes/configure_linux_riscv.mk

```diff
@@ -21,14 +21,17 @@
 
 include $(CONFIG_INCL_DIR)/configure_common.mk
 
-RISCV_CROSSTOOLS_PREFIX=riscv64-unknown-linux-gnu
-
 ifneq (,$(findstring _riscv64, $(SPEC)))
-	RISCV_TARGET=riscv64-unknown-linux-gnu
+	OMR_RISCV_CROSSTOOLS_PREFIX=$(RISCV_CROSSTOOLS_PREFIX)
 else
-	RISCV_TARGET=riscv32-unknown-linux-gnu
+	OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-unknown-linux-gnu
+	ifeq (install, $(RISCV_TOOLCHAIN_TYPE))
+		OMR_RISCV_CROSSTOOLS_PREFIX=riscv32-linux-gnu
+	endif

```

Comment:
These lines should not begin with tabs. I suggest the default should be in the else branch:
```
  ifeq (install, $(RISCV_TOOLCHAIN_TYPE))
    OMR_RISCV_CROSSTOOLS_PREFIX := riscv32-linux-gnu
  else
    OMR_RISCV_CROSSTOOLS_PREFIX := riscv32-unknown-linux-gnu
  endif
```

Reply:
Updated the script as suggested above.